Palindrome/palindrome_no.java

```diff
@@ -0,0 +1,44 @@
+// Java program to check if a number entered is a palindrome
+
+import java.util.*;
+
+public class palindrome_no {
+	
+	// Find reverse of a number
+	public static int reverse(int number) {
```

Comment:
use 4 space indentation instead of using tabs

Reply:
@raghav-dalmia  I have made the changes. Plz review